tardis/plasma/tests/test_write_to_dot.py

```diff
@@ -0,0 +1,27 @@
+import tardis
+from tardis import run_tardis
+from tardis.io.atom_data.util import download_atom_data
+import os
+
+import tempfile
+
+#DATA_PATH = os.path.join(tardis.__path__[0], 'plasma', 'tests', 'data')
+
+def run_simulation(yml_file):
```

Comment:
this is already done somewhere else in the tests - have a look at pytest and fixtures